packages/flutter_tools/test/general.shard/dart/pub_get_test.dart

```diff
@@ -512,223 +498,6 @@ void main() {
-      expect(logger.statusText,
-        'Running "flutter pub get" in /...\n'
-        'pub get failed (server unavailable) -- attempting retry 1 in 1 second...\n'
-        'pub get failed (server unavailable) -- attempting retry 2 in 2 seconds...\n'
-        'pub get failed (server unavailable) -- attempting retry 3 in 4 seconds...\n'
-        'pub get failed (server unavailable) -- attempting retry 4 in 8 seconds...\n'
-        'pub get failed (server unavailable) -- attempting retry 5 in 16 seconds...\n'
-        'pub get failed (server unavailable) -- attempting retry 6 in 32 seconds...\n'
-        'pub get failed (server unavailable) -- attempting retry 7 in 64 seconds...\n'
-        'pub get failed (server unavailable) -- attempting retry 8 in 64 seconds...\n' // at t=39
-        'pub get failed (server unavailable) -- attempting retry 9 in 64 seconds...\n' // at t=103
-        'pub get failed (server unavailable) -- attempting retry 10 in 64 seconds...\n', // at t=167
-      );
-    });
-    expect(logger.errorText, isEmpty);
-    expect(error, isNull);
-    expect(processManager, hasNoRemainingExpectations);
-  });
-
-  testWithoutContext('pub get offline does not retry', () async {
-    String? error;
-
-    const FakeCommand pubGetCommand = FakeCommand(
-      command: <String>[
-          'bin/cache/dart-sdk/bin/dart',
-          '__deprecated_pub',
-          '--directory',
-          '.',
-          'get',
-          '--offline',
-          '--example',
-      ],
-      exitCode: 69,
-      environment: <String, String>{'FLUTTER_ROOT': '', 'PUB_ENVIRONMENT': 'flutter_cli:flutter_tests'},
-    );
-    final FakeProcessManager processManager = FakeProcessManager.list(<FakeCommand>[
-      pubGetCommand,
-    ]);
-    final BufferLogger logger = BufferLogger.test();
-    final FileSystem fileSystem = MemoryFileSystem.test();
-    final Pub pub = Pub(
-      fileSystem: fileSystem,
-      logger: logger,
-      processManager: processManager,
-      usage: TestUsage(),
-      platform: FakePlatform(),
-      botDetector: const BotDetectorAlwaysNo(),
-      stdio: FakeStdio(),
-    );
-
-    FakeAsync().run((FakeAsync time) {
-      expect(logger.statusText, '');
-      pub.get(
-        project: FlutterProject.fromDirectoryTest(fileSystem.currentDirectory),
-        context: PubContext.flutterTests,
-        offline: true
-      ).then((void value) {
-        error = 'test completed unexpectedly';
-      }, onError: (dynamic thrownError) {
-        error = 'test failed unexpectedly: $thrownError';
-      });
-      time.elapse(const Duration(milliseconds: 500));
-      expect(logger.statusText,
-        'Running "flutter pub get" in /...\n'
-      );
-    });
-    expect(logger.errorText, isEmpty);
-    expect(error, contains('test failed unexpectedly: Exception: pub get failed'));
-    expect(processManager, hasNoRemainingExpectations);
-  });
-
-  testWithoutContext('pub get 66 shows message from pub', () async {
```

Comment:
Test no longer possible because of the "pub get" command adopting ProcessStartMode.inheritStdio, which does not allow you to mock stdout/stderr.